src/zarr/store/remote.py

```diff
@@ -205,5 +205,6 @@ async def list_dir(self, prefix: str) -> AsyncGenerator[str, None]:
             yield onefile
 
     async def list_prefix(self, prefix: str) -> AsyncGenerator[str, None]:
-        for onefile in await self._fs._ls(prefix, detail=False):
-            yield onefile
+        find_str = "/".join([self.path, prefix])
+        for onefile in await self._fs._find(find_str):
```

Comment:
The defaults for find are: maxdepth=None, withdirs=False, detail=False; maybe good to be specific.

Why is find() better than ls()? The former will return all  child files, not just one level deep - is that the intent? If not, ls() ought to be generally more efficient.

Reply:
using `find` here is merely due to my ignorance of fsspec. I will implement `ls` as you suggest

Reply:
It depends on whether you want one directory level or everything below it. When I wrote the original, I didn't know the intent.

Reply:
i believe the intent here is to list everything below `prefix` (at least, that's how I'm using it)

Reply:
I misunderstood your first comment. since the intent is to use the behavior of `_find`, I'm keeping it, but adding explicit kwargs as you suggested.